Makefile

```diff
@@ -103,7 +104,7 @@ LLD_LIBS = $(START_GROUP) -llldCOFF -llldCommon -llldCore -llldDriver -llldELF -
 # For static linking.
 ifneq ("$(wildcard $(LLVM_BUILDDIR)/bin/llvm-config*)","")
     CGO_CPPFLAGS=$(shell $(LLVM_BUILDDIR)/bin/llvm-config --cppflags) -I$(abspath $(LLVM_BUILDDIR))/tools/clang/include -I$(abspath $(CLANG_SRC))/include -I$(abspath $(LLD_SRC))/include
-    CGO_CXXFLAGS=-std=c++11
+    CGO_CXXFLAGS+=-std=c++11
```

Comment:
(related to the above comment)

Reply:
Yeah, you are right. And these problems seem to have been fixed with LLVM10 (it now seems to know about the weird placement of /usr/include on a mac)